### ISSUE TITLE
Save using member resource id

### ISSUE THREAD

**Issue by coatezy** [dwbutler/groupify#33]
Hey.

I was wondering if there is a way to save group members by there id's (collection_singular_ids)? For example I'd like to have a multi select of members in my group form.

```
f.input :user_ids, as: :select, collection: User.all, input_html: { multiple: true }

```
```
Group.find(1).user_ids = [1, 2, 3]
```
But I am finding the group_type is not being set (default STI behaviour) so no results as returned when queried as it expects group_type to be set to Group.

If I use
```
 Group.find(1).add(User.find(1)) 
```

The group_type is set on the group_membership.

Cheers,
Tom

**Comment by dwbutler** [comment]
This might be caused by the associations being set up incorrectly. Will investigate.

**Comment by coatezy** [comment]
@dwbutler Thanks! If I get a chance to take a closer look and find the problem I'll submit a PR.

**Comment by dwbutler** [comment]
That would be awesome.

**Comment by dwbutler** [comment]
@coatezy I tried this in my app and think I found the problem. Can you try out #34?

**Comment by coatezy** [comment]
@dwbutler Just switched to the branch and tested. Now working as expected. Thanks! :smile:

**Comment by dwbutler** [comment]
Awesome. I'll add some specs for the change and release it soon.

**Comment by dwbutler** [comment]
I wasn't able to find time to write specs for this fix, but I went ahead and released it in 0.7.1. Sorry for the delay.
